cleverhans/attacks_tf.py

```diff
@@ -16,31 +16,55 @@
 FLAGS = flags.FLAGS
 
 
-def fgsm(x, predictions, eps, clip_min=None, clip_max=None):
+def fgsm(x, predictions, eps=0.3, clip_min=None, clip_max=None):
+    warnings.warn("attacks_tf.fgsm is deprecated and will be removed on "
```

Comment:
That's fair, I removed the deprecate warnings from the tf and th implementations in 0b1a2e5